index.html

```diff
@@ -0,0 +1,68 @@
+                  <div class="navbar-item">Store</div>
+                  <div class="navbar-item">Mac</div>
+                  <div class="navbar-item">iPod</div>
+                  <div class="navbar-item">iPhone</div>
+                  <div class="navbar-item">iPad</div>
+                  <div class="navbar-item">iTunes</div>
+                  <div class="navbar-item">Support</div>
+                  <div class="search-bar"><img class="search-icon" src="./images/search.svg" alt="search icon" /><input type="search" name="search" id="search"></div>
+              </nav>
+            <div class="header-text">
+                <p class="big-text">What will your verse be?</p>
+                <p class="medium-text">Everyone has something to share. See how poeple are using iPad in their unique ways to add world's story.</p>
+                <p class="small-text"><a href="#">Watch the "Your Verse" film and explore the stories</a> <i class="fa fa-play-circle-o" aria-hidden="true"></i>  </p>
+            </div>
+          </div>
+        </div>
+
+        <div class="photo1">
+          <div class="cell">
+            <a href="#"><img class="photo" src="./images/iphone1.jpg" alt="Iphone 5s"></a>
+          </div>
+        </div>
+        <div class="photo2">
+          <div class="cell">
+            <a href="#"><img class="photo" src="./images/iphone2.jpg" alt="Iphone 5c"></a>
+          </div>
+        </div>
+        <div class="photo3">
+          <div class="cell">
+            <a href="#"><img class="photo" src="./images/apple1.jpg" alt="Mac birthday"></a>
+          </div>
+        </div>
+        <div class="photo4">
+          <div class="cell">
+            <a href="#"><img class="photo" src="./images/apple2.jpg" alt="Supplier Responsability"></a>
+          </div>
+        </div>
+        <div class="footer1">
+          <div class="cell info-left">
+            <p class="info">Shop the <a class="info-link" href="#">Apple Online Store</a> (1-800-MY-APPLE), visit an <a class="info-link" href="#">Apple Retail Store</a>, or find a <a class="info-link" href="#">reseller</a>.</p>
+            <p class="info">Copyright © 2014 Apple Inc. All rights reserved. <a class="info-link" href="#">Terms of Use</a> | <a class="info-link" href="#">Privacy Policy</a></p>
+          </div>
+        </div>
+        <div class="footer2">
+          <div class="cell info-right">
+            <p class="info"> <a class="info-link" href="#">Site Map</a> | <a class="info-link" href="#">Hot News</a> | <a class="info-link" href="#">RSS Feeds</a> | <a class="info-link" href="#">Media Info</a> | <a class="info-link" href="#">Environment</a> | <a class="info-link" href="#">Job Opportunities</a> | <a class="info-link" href="#">Contact Us</a> | <a class="info-link" href="#"><img class="little-image" src="./images/usa.png" alt="country"></a></p>
+          </div>
+        </div>
```

Comment:
- [ ] [MANDATORY] The footer should be wrapped by `<footer>` tags.

- [ ] [MANDATORY] The links should look like those in [the original page](https://web.archive.org/web/20140301004610/http://www.apple.com/).

Original Page          |  Your Page
:-------------------------:|:-------------------------:
![Screen Shot 2020-07-17 at 11 19 38 PM](https://user-images.githubusercontent.com/18187039/87837724-072f4c80-c884-11ea-9480-87913692bb93.png) |  ![Screen Shot 2020-07-17 at 11 18 36 PM](https://user-images.githubusercontent.com/18187039/87837680-e1a24300-c883-11ea-808a-110d7c9bffc2.png)

- [ ] [MANDATORY] The web page should take the whole width of the screen as seen in [the original page](https://web.archive.org/web/20140301004610/http://www.apple.com/).